network/src/inbound/inbound.rs

```diff
@@ -158,14 +157,15 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
                 metrics::increment_counter!(inbound::GETBLOCKS);
 
                 if self.sync().is_some() {
+                    let hashes = hashes.into_iter().map(|x| x.0.into()).collect();
```

Comment:
If I'm not mistaken, these are conversions from `BlockHeaderHash` to `Digest`? It would be nice to have an `impl From<BlockHeaderHash> for Digest` to make this a bit simpler.

Reply:
i'll gladly take a `.0` for now to avoid coupling digest to snarkvm any more